extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/quantiles/sql/DoublesSketchListArgBaseOperatorConversion.java

```diff
@@ -144,7 +143,7 @@ public boolean checkOperandTypes(SqlCallBinding callBinding, boolean throwOnFail
 
         // Verify that 'operand' is a literal number.
         if (!SqlUtil.isLiteral(operand)) {
```

Comment:
doesn't this need to allow cast?

Reply:
Good catch, it does. I modified the call to allow casts and added test for the same